server/routes/users.js

```diff
@@ -1,9 +1,39 @@
-var express = require('express');
-var router = express.Router();
+const express = require('express');
+const router = express.Router();
 
 /* GET users listing. */
 router.get('/', function(req, res, next) {
   res.send('respond with a resource');
```

Comment:
why does this temp string say "resource"? Should be user? Is the path correct here or should it be /users?